spec/e2e/support/target-blank.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @typedef {Pick<HTMLAnchorElement, "innerText" | "href" | "target">} SimplifiedLink
+ */
+
+/**
+ * @param {import('puppeteer').Page} page
+ * @return {Promise<SimplifiedLink[]>}
+ */
+async function getLinks(page) {
+  return await page.$$eval('a', (aTags) =>
```

Comment:
There's some [subtle differences](https://jakearchibald.com/2017/await-vs-return-vs-return-await/), but the `async` / `await` here is arguably unnecessary. This is the same:

```suggestion
function getLinks(page) {
  return page.$$eval('a', (aTags) =>
```

Reply:
😩 sometimes I miss the ES5 days when there was less syntax to learn .... all I wanted was the arrow function and null navigation from coffee....

Reply:
But also TIL so thank you

Reply:
> 😩 sometimes I miss the ES5 days when there was less syntax to learn .... all I wanted was the arrow function and null navigation from coffee....

Sure beats [callback hell](http://callbackhell.com/) when you actually _do_ need the `await` 🤷